storybook/src/docs/components/AppIcons.tsx

```diff
@@ -8,7 +8,7 @@ import { Image } from '@amsterdam/design-system-react'
 export const AppleTouchIcon = () => (
   <div className="ams-docs-gallery">
     <figure className="ams-docs-figure">
-      <Image alt="" height={180} src="favicon/apple-touch-icon.png" width={180} />
+      <Image alt="" className="ams-aspect-ratio-1-1" height={180} src="favicon/apple-touch-icon.png" width={180} />
```

Comment:
`Image` has an `aspectRatio` prop for this.

Reply:
Check, I thought we opted for just the util class.